react-native/react/menubar/index.js

```diff
@@ -40,12 +41,20 @@ class Menubar extends Component {
     const onMenubarShow = () => {
       setImmediate(() => {
         this.checkForFolders()
+
+        engine.listenOnConnect('menubar', () => {
```

Comment:
oh i guess registering will call it no matter what so it always calls, k lemme branch again